### ISSUE TITLE
Popup to restart VScode after install

### ISSUE THREAD

**Issue by ZobelOmry** [strigo/vscode-linear-extension#7]
If user doesnt restart vscode there is a bug "title cannot be empty" or something when creating an issue

**Comment by nir0s** [comment]
We should indeed do that. It's currently stated in the docs, but there's no reason not to add it as an action.

**Comment by guzzur** [comment]
Not sure why we need to restart vscode. Rather, let's fix the issue why this happens in the first place.

**Comment by nir0s** [comment]
Indeed. I saw that vscode no longer requires a reload post extension installation, so we may as well fix.
